src/Adapters/Auth/apple.js

```diff
@@ -32,15 +32,26 @@ const getApplePublicKey = async () => {
   return currentKey;
 };
 
-const verifyIdToken = async ({ token, id }, clientID) => {
+const verifyIdToken = async ({ token, id }, clientIDs) => {
   if (!token) {
     throw new Parse.Error(
       Parse.Error.OBJECT_NOT_FOUND,
       'id token is invalid for this user.'
     );
   }
   const applePublicKey = await getApplePublicKey();
-  const jwtClaims = jwt.verify(token, applePublicKey, { algorithms: 'RS256' });
+  let jwtClaims;
+  // this throws an error if token has expired, catching so that we can
+  // throw a parse error
```

Comment:
Remove this comment. It can throw a variety of errors. We can see it throws a parse error.